### ISSUE TITLE
UpsertSignature: better control over ON CONFLICT DO UPDATE

### ISSUE THREAD

**Issue by avnersch** [jawj/zapatos#59]
https://jawj.github.io/zapatos/#upsert
`UpsertSignature` accepts these arguments:
```typescript
interface UpsertSignatures {
  <T extends Table, C extends ColumnForTable<T>[] | undefined, E extends SQLFragmentsMap | undefined>(
    table: T,
    values: InsertableForTable<T>,
    conflictTarget: UpsertConflictTargetForTable<T>,
    options?: UpsertOptions<T, C, E>
  ): SQLFragment<UpsertReturnableForTable<T, C, E>>;
```

 It populates the ON CONFLICT DO UPDATE part, by taking the columns from the insert clause (`values: InsertableForTable<T>`) and excluding from them the conflict target columns (`conflictTarget: UpsertConflictTargetForTable<T>`)
 
 For my use case, I'd like to specify a smaller subset of columns to update on conflict.
 
 Would it be possible to add an optional argument of column names array, to be used for the update part?
 
 Thanks for the lib,
 Avner

**Comment by jawj** [comment]
I've had a go at this by adding an `updateColumns` option to `upsert` in 60df7616cf68aeb8217675a8badf53c5be0a1284, but I haven't tested it. 

Would you mind seeing if it does what you need?

You'll need to `git clone` this repo, `npm run install`, `npm run build`, `npm link`, and then from your calling project `npm link zapatos`.

**Comment by jawj** [comment]
I tested (and fixed!) this, and it's released in 3.3.

**Comment by avnersch** [comment]
@jawj Much appreciated, thank you!
